creds.go

```diff
@@ -195,3 +196,25 @@ func (creds *Credentials) Cert() (cert *Certificate, err error) {
 	}
 	return
 }
+
+// LoadCredentials attempts to load a set of Taskcluster Credentials from the
+// environment using the standard Taskcluster environment variables.  This
+// function never fails, but might return a pointer to a nil
+// tcclient.Credentials struct.
+func LoadCredentials() *Credentials {
```

Comment:
Maybe we should call this `CredentialsFromEnvVars` or something similar, so it is clear credentials are initialised based on environment variables?